src/main/java/de/tum/in/www1/artemis/domain/Result.java

```diff
@@ -228,6 +228,10 @@ public void setScore(Long score) {
     }
 
     public Boolean isRated() {
+        if (rated == null) {
```

Comment:
Please remove this code again. We previously explicitly deactivated this, because we distinguish all three cases: null, Boolean.False and Boolean.True. To prevent null pointer exceptions, please use `if(result.isRated() == Boolean.TRUE)` instead of `if(result.isRated())`

Reply:
Done